### ISSUE TITLE
MDL checkboxes don't work with column-count

### ISSUE THREAD

**Issue by kengruven** [google/material-design-lite#4030]
I made a page with a few `mdl-checkbox` instances, and then wrapped them in a `<div style="column-count: 2">` to make two nice columns.  They all look fine, but the checkboxes in the second column don't work quite right.

Here's an example: http://codepen.io/anon/pen/eJMxpQ

In Safari 9, I get 3 checkboxes in the first column, and 1 checkbox in the second column.  The first 3 checkboxes work great.  The one in the second column is funny.  The border of the box does change from gray to blue and back, but it never fills.  When you click the box, the "touch" circle appears in both the bottom of the first column, and in the second column.

<img width="730" alt="mdl-checkbox-column-count" src="https://cloud.githubusercontent.com/assets/22565/12600587/cbc448e0-c44f-11e5-8db6-2dd35064e7cb.png">

In Firefox, when using the `-moz-` prefix, they work great.

In Chrome, when using the `-webkit-` prefix, they are broken in the same way as Safari, except that the "touch" circle never appears at all.

I haven't tried Explorer yet.

**Comment by surma** [comment]
Very interesting bug report, thank you @kengruven. I don’t think we have tested against `column` styles at all. Not sure if we can/will fix this in the MDL 1 timeline, as we are about to start working towards v2, but we will definitely keep this in mind.

@sgomes @addyosmani WDYT?

**Comment by Garbee** [comment]
It is also worth noting that [columns still have bugs](https://code.google.com/p/chromium/issues/detail?id=530074) so this could be stumbling into the same kind of territory.

Deeper investigation is needed here. Also, it should certainly not be something to look at in 1.x as far as a blocking bug since the column specification is still not a candidate recommendation. It is development technology and people building with it should understand that. We should keep it in mind when building for 2.x, but remembering it may still have bugs itself within engines.

**Comment by sgomes** [comment]
Agreed, this would be 2.0 work if anything.

**Comment by kengruven** [comment]
Yeah, I didn't realize `column-count` was still so experimental.  I saw an unprefixed selector, and I checked caniuse.com and didn't see any issues related to what I was seeing, so I assumed it should be safe.

If this isn't actually MDL-related, then feel free to close this issue.
